djautotask/models.py

```diff
@@ -647,6 +652,9 @@ class Task(ATUpdateMixin, TimeStampedModel):
     department = models.ForeignKey(
         'Department', blank=True, null=True, on_delete=models.SET_NULL
     )
+    task_category = models.ForeignKey(
```

Comment:
Change this to just `category`. We already know its a task category, and if it has the same field name as ticket categories that will help with generalization.